pymodbus/client/async/asyncio/__init__.py

```diff
@@ -707,11 +700,18 @@ def connect(self):
         """
         _logger.debug('Connecting.')
         try:
-            yield from self.loop.create_connection(self._create_protocol)
-            _logger.info('Connected to %s:%s.' % (self.host, self.port))
+            from serial_asyncio import create_serial_connection
```

Comment:
Please move this change to serial factory, https://github.com/cbergmiller/pymodbus/blob/fix/serial_async_client/pymodbus/client/async/factory/serial.py#L82.

Reply:
Thanks for the review. I updated the pull request.